presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -194,6 +199,15 @@ public void setCursorName(String name)
         // ignore: positioned modifications not supported
     }
 
+    private Map<String, String> getStatementSessionProperties() {
```

Comment:
We normally put private methods at the end of the class (after the public methods)